homeassistant/components/vallox/sensor.py

```diff
@@ -127,6 +130,24 @@ def native_value(self) -> StateType:
         return VALLOX_CELL_STATE_TO_STR.get(super_native_value)
 
 
+class ValloxProfileDurationSensor(ValloxSensorEntity):
+    """Child class for profile duration reporting."""
+
+    @property
+    def native_value(self) -> StateType:
+        """Return the value reported by the sensor."""
+minutes_left: int | float | None = self.coordinator.data.get({
+    Profile.EXTRA: "A_CYC_EXTRA_TIMER",
+    Profile.FIREPLACE: "A_CYC_FIREPLACE_TIMER",
+    Profile.BOOST: "A_CYC_BOOST_TIMER"
+}.get(self.coordinator.data.profile), 0)
```

Comment:
Oops this should fix indentation

```suggestion
        minutes_left: int | float | None = self.coordinator.data.get({
            Profile.EXTRA: "A_CYC_EXTRA_TIMER",
            Profile.FIREPLACE: "A_CYC_FIREPLACE_TIMER",
            Profile.BOOST: "A_CYC_BOOST_TIMER"
        }.get(self.coordinator.data.profile), 0)
```

Reply:
the types don't match either so i just reverted it